src/libtomahawk/Config.h.in

```diff
@@ -15,16 +15,18 @@
 
 #cmakedefine WITH_CRASHREPORTER
 #cmakedefine WITH_BINARY_ATTICA
-#cmakedefine WITH_QtSparkle
 #cmakedefine WITH_UPOWER
 #cmakedefine WITH_GNOMESHORTCUTHANDLER
 
 #cmakedefine LIBLASTFM_FOUND
 #cmakedefine QCA2_FOUND
+#cmakedefine QtSparkle_FOUND
```

Comment:
No. WITH and FOUND is different. WITH can be manually switched off. Why did you change it?